### PR TITLE
Fix github templates and devdoc links

### PR DESCRIPTION
**Pull request by MeKeyCool**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update some comments and files for PR process and devdoc links
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Na.
| How to test?      | Na.
| Possible impacts? | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28040)

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: I need help
-    url: https://devdocs.prestashop.com/1.7/faq/i-need-help/
+    url: https://devdocs.prestashop.com/8/faq/i-need-help/
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/config.yml]: I think this link makes more sense
```suggestion
    url: https://www.prestashop-project.org/support/
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requ
 | Category?         | FO / BO / CO / IN / WS / TE / LO / ME / PM
 | BC breaks?        | yes / no
 | Deprecations?     | yes / no
-| Fixed ticket?     | Fixes #{issue number here}.
+| Fixed ticket?     | Fixes #{issue number here}. If no issue / feature created, please consider creating one for QA purpose.
```
Comment (eternoendless) [inline .github/PULL_REQUEST_TEMPLATE.md]: The issue is only required for bug fixes and new features

**Review by eternoendless — COMMENTED** [review]
I don't know if it's the right time to link to v8 doc yet, as it's still in development and not the canonical link (hence the notice at the top of all pages in v8 doc)

**Comment by MeKeyCool** [comment]
> canonical link

Develop branch will be tagged under v8. So i thought it is consistent to have v8 documentation linked to this branch.
In fact, i think it should be updated every time your create a new stable version branch.

```mermaid
graph BT
    title[develop] --- A(( ))
    linkStyle 0 stroke:blue;
    A --- A1(<i>Doc X.Y</i>)
    A --- B(( ))
    linkStyle 2 stroke:blue;
    B --> B1[X.Y branch]
    linkStyle 3 stroke:blue;
    B --- C(( ))
    linkStyle 4 stroke:green;
    C --- C1(<i>Doc X.Y+1</i>)
    C --- D(( ))
    linkStyle 6 stroke:green;
    D --> D1[X.Y+1 branch]
    linkStyle 7 stroke:green;
    D --- E(( ))
    linkStyle 8 stroke:yellow;
    E --- E1(<i>Next doc</i>)
    E --> F[next branch & develop]
    linkStyle 10 stroke:yellow;
```

**Review by matks — COMMENTED** [review]
@MeKeyCool 

> Develop branch will be tagged under v8. So i thought it is consistent to have v8 documentation linked to this branch.

The problem is that people open https://github.com/prestashop/prestashop and see the README. As very few people use `develop` locally, we can guess most visitors of the GitHub homepage are using PS 1.7. So they might be looking for guidance about PS 1.7 .

Yes, the `develop` branch will become 8.0.0 . But `develop` is the defaut branch so the README displayed in the one from `develop` and for the majority of users, PS 8 is a work-in-progress while they use 1.7 in production.

That being said I ❤️ the changes you submitted

**Comment by MeKeyCool** [comment]
> @MeKeyCool
> 
> > Develop branch will be tagged under v8. So i thought it is consistent to have v8 documentation linked to this branch.
> 
> The problem is that people open https://github.com/prestashop/prestashop and see the README. As very few people use `develop` locally, we can guess most visitors of the GitHub homepage are using PS 1.7. So they might be looking for guidance about PS 1.7 .
> 
> Yes, the `develop` branch will become 8.0.0 . But `develop` is the defaut branch so the README displayed in the one from `develop` and for the majority of users, PS 8 is a work-in-progress while they use 1.7 in production.
> 
> That being said I heart the changes you submitted

Then if the goal that random people coming to https://github.com/prestashop/prestashop are redirected to current stable version, i suggest to consider project default branch : https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch

The problem of this solution is : we can't have a default behavior for both target "random users" and "developer community".

So if the solution is to keep current stable version documentation link in README (which is almost home page of the project), Why not assuming it properly with a dedicated section ?
For example, creating an `About the 'stable' branch` like you did a `[About the 'develop' branch](https://github.com/PrestaShop/PrestaShop#about-the-develop-branch)`

Furthermore, i didn't update the README xD !
It looks already updated to v8. But I can see other urls, `https://docs.prestashop-project.org/1.7-documentation/user-guide` and `https://docs.prestashop-project.org/1-6-documentation/`, that I didn't check for update.

Should I check them ? Should I reset README to v1.7 ?

**Comment by matks** [comment]
> For example, creating an About the 'stable' branch like you did a [About the 'develop' branch](https://github.com/PrestaShop/PrestaShop#about-the-develop-branch)

I find this idea interesting. This solves our issue nicely: we clearly provide informations about 1.7 to 1.7 visitors and information about develop to develop visitors.

Wdyt @PrestaShop/prestashop-maintainers

**Review by jolelievre — APPROVED** [review]
@eternoendless @matks 

I don't see the problem of changing the doc links to v8, since it's the most recent

The develop branch is indeed closer to v8 than to 1.7 anyway, so if we really want to distinguish between v8 and develop then I guess we would need a documentation for develop branch? Until then v8 doc seems like the best candidate to me

**Comment by jolelievre** [comment]
And if the stable readme reassures you no problem for me anyway ^^

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @MeKeyCool

**Comment by kpodemski** [comment]
@Progi1984 I think this one has not been decided yet 🤔

**Comment by Progi1984** [comment]
@kpodemski Sorry, I didn't see any "Request Changes" so no blocking changes.

**Pull request merged**